src/libYARP_dev/src/devices/FrameTransformServer/FrameTransformServer.cpp

```diff
@@ -432,7 +432,56 @@ bool FrameTransformServer::threadInit()
     return true;
 }
 
+bool FrameTransformServer::parseStartingTf(yarp::os::Searchable &config)
+{
+    if (config.find("starting_tf").isInt())
```

Comment:
This is better:
yarp::os::Searchable& tf_group = config.findGroup("STARTING_TF");

BTW: I'd suggest different group name, such as USER_TF, STATIC_TF etc..